README.md

```diff
@@ -136,6 +136,119 @@ rclip -p kitty
   ```
 </details>
 
+### Advanced Options
+
+#### Custom Database Location
+
+By default, rclip stores its database in your system's application data directory. You can specify a custom location:
+
+```bash
+# Use a custom database file
+rclip --db-path /path/to/my/photos.db "sunset"
+
+# Store database on an external drive
+rclip --db-path /mnt/photos/rclip.db "vacation"
+```
+
+#### Search in Different Directories
+
+Search for images in a specific directory without changing your current directory:
+
+```bash
+# Search in a different directory
+rclip --search-dir /path/to/photos "cats"
+
+# Search external drive
+rclip --search-dir /mnt/backup/photos "birthday"
+```
+
+#### Index-Only Mode
+
+Index images without performing a search - useful for pre-building the database:
+
+```bash
+# Just index the current directory
+rclip --index-only
+
+# Index a specific directory
+rclip --index-only --search-dir /path/to/new/photos
+
+# Index to a custom database
+rclip --index-only --db-path /path/to/custom.db --search-dir /photos
+```
```

Comment:
This is nice, but I am not sure if we should merge it as-is because how it's implemented confuses the rclip `--help` message.

Please see my comment [here](https://github.com/yurijmikhalevich/rclip/pull/190#discussion_r2280699745).